packages/cli/test/utils.js

```diff
@@ -116,4 +117,116 @@ describe('Utils', () => {
       });
     }
   });
+  describe('findArtifactPaths', () => {
+    it('returns all matching paths of type', () => {
+      const expected = [
+        path.join('tmp', 'app', 'models', 'foo.model.js'),
+        path.join('tmp', 'app', 'models', 'baz.model.js'),
+      ];
+
+      const reader = () => {
+        // Add the expected values to some extras.
+        const result = [
+          path.join('tmp', 'app', 'models', 'bar.js'),
+          path.join('tmp', 'app', 'models', 'README.js'),
+        ].concat(expected);
+        return Promise.resolve(result);
```

Comment:
Why is it necessary to wrap the result in a promise? Can we fix `utils.findArtifactsPaths` to call `Promise.resolve` for us?